docker/Dockerfile.amd

```diff
@@ -82,26 +82,8 @@ RUN cd /tmp/ \
     && make -j ${nproc} \
     && make install
 
-# Install OpenMPI
-RUN cd /tmp \
-    && git clone --recursive https://github.com/open-mpi/ompi.git -b ${OMPI_BRANCH} \
-    && cd ompi \
-    && ./autogen.pl \
-    && mkdir build \
-    && cd build \
-    && ../configure --prefix=$OMPI_HOME --with-ucx=$UCX_HOME \
-        --enable-mca-no-build=btl-uct  \
-        --without-verbs \
-        --with-pmix  \
-        --enable-mpi \
-        --enable-mpi-fortran=yes \
-        --disable-debug \
-    && make -j ${nproc} \
-    && make install
-
 # Cleanup
-RUN rm -rf /tmp/ompi && rm -rf /tmp/ucx
-
+RUN rm -rf /tmp/ucx
 
 # Adding OpenMPI and UCX to Environment
 ENV PATH=$OMPI_HOME/bin:$UCX_HOME/bin:$PATH \
```

Comment:
You have removed OpenMPI but it's still present in the path.

Reply:
I didn't remove open MPI I just moved it below with each compiler but the path is still the same for both images